data-acquisition/nomask_generator/No_mask_downloader.ipynb

```diff
@@ -0,0 +1,170 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# Downloading and Generating Masks for Non Class Examples"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 45,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from PIL import Image\n",
+    "from google_images_download import google_images_download\n",
+    "import Augmentor\n",
+    "import glob\n",
+    "import shutil\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 46,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def create_masks(path):\n",
+    "    \"\"\"\n",
+    "    This function generate empty black masks for each of \n",
+    "    the images downloaded from Google. \n",
+    "    \"\"\"\n",
+    "    x = path.rstrip('\\n')\n",
+    "    t = x.split('/')[-1].rstrip('.jpg')\n",
+    "    im = Image.open(x)\n",
+    "    im_size = im.size\n",
```

Comment:
Is this really necessary? Why can't we just use `im.size`?

Reply:
Alright, I've made some changes, including modularizing the code so it can be imported into notebook, and also removing augmentation.